features/product/managing_product_variants/sorting_product_variants_within_product_by_position.feature

```diff
@@ -12,44 +12,58 @@ Feature: Sorting listed product variants from a product by position
         And this product has also an "Opel Insignia Sedan" variant at position 1
         And I am logged in as an administrator
 
-    @ui
+    @ui @api
     Scenario: Product variants are sorted by position in ascending order by default
         When I view all variants of the product "Opel Insignia"
         Then I should see 3 variants in the list
         And the first variant in the list should have name "Opel Insignia Hatchback"
         And the last variant in the list should have name "Opel Insignia Sports Tourer"
 
-    @ui
+    @ui @api
     Scenario: Sorting product variants in descending order
         When I view all variants of the product "Opel Insignia"
         And I start sorting variants by position
         Then the first variant in the list should have name "Opel Insignia Sports Tourer"
         And the last variant in the list should have name "Opel Insignia Hatchback"
 
-    @ui
+    @ui @api
     Scenario: New product variant with no position is added as the last one
         Given the product "Opel Insignia" has also an "Opel Insignia Country Tourer" variant
         When I view all variants of the product "Opel Insignia"
         Then I should see 4 variants in the list
         And the last variant in the list should have name "Opel Insignia Country Tourer"
 
-    @ui
+    @ui @api
     Scenario: New product variant with position 0 is added as the first one
         Given the product "Opel Insignia" has also an "Opel Insignia Country Tourer" variant at position 0
         When I view all variants of the product "Opel Insignia"
         Then I should see 4 variants in the list
         And the first variant in the list should have name "Opel Insignia Country Tourer"
 
-    @ui @javascript
+    @ui @javascript @no-api
     Scenario: Setting product variant as the first one in the list
         When I view all variants of the product "Opel Insignia"
         And I set the position of "Opel Insignia Sedan" to 0
         And I save my new configuration
-        And the first variant in the list should have name "Opel Insignia Sedan"
+        Then the first variant in the list should have name "Opel Insignia Sedan"
 
-    @ui @javascript
+    @api @no-ui
```

Comment:
I've a limited experience with api .features, but I don't remember seeing technique of identical scenarios, but with slightly different phrasing for @ui/@api before.

Maybe api context can be adjusted to handle origianal scenario somehow?

@Sylius/development-team please advise.

Reply:
From my side, making it "just to conform steps happening on the UI" seemed stupid and required a lot of additional work inside the Behat steps impl.

Reply:
As much as possible, it would be good to have one scenario for both API and UI, but not at all costs, because sometimes this behaviour is a bit different. We already have some cases like this where we have separated scenarios depending on the context.